### ISSUE TITLE
Bug: Data for contributors list not adding up

### ISSUE THREAD

**Issue by BekahHW** [open-sauced/app#2327]
### Describe the bug

<img width="1053" alt="image" src="https://github.com/open-sauced/app/assets/34313413/efa83e6a-72ec-4e15-831a-22a396e14462">

I'm not sure if this is an API issue or not. But if you look here, this contributor has a "medium" activity score. However, it says the last contribution was 11 _years_ ago. If you look further down the row, we have an activity graph that shows a recent contribution. 

This contributor graph from their profile indicates a recent activity:

<img width="682" alt="image" src="https://github.com/open-sauced/app/assets/34313413/8e49ebd5-a59a-46d7-9e0a-5eebfc17f434">

Based on that, I'm inclined to believe that the 11 years ago isn't updating at the same rate as the activity chart and level, but we need that to be the case.

### Steps to reproduce

1. Create a list
2. look at https://app.opensauced.pizza/user/lopter
3. See the discrepancy.

### Browsers

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by BekahHW** [comment]
@open-sauced/engineering any ideas here?

**Comment by jpmcb** [comment]
Yeah, i think there's a data discrepancy in how the app consumes the info we have - worth a deep dive but this definitely gets better with v2 API and the real time events data

**Comment by OgDev-01** [comment]
I agree with john. Since v2 API migrations is already in flight, we might as well fix the discrepancy during the migration.

**Comment by BekahHW** [comment]
There's still some disconnect here, even though it's not as noticeable. This is from Apr 4:

[Insight](https://app.opensauced.pizza/workspaces/0994f662-b90f-43f1-b402-eadaf1d4643f/contributor-insights/ccec6984-c62e-4470-9b8b-8c2dd724d355/overview)
<img width="360" alt="image" src="https://github.com/open-sauced/app/assets/34313413/944cd734-f1ba-4027-9019-ff1d1a9d2ce5">

[profile](https://app.opensauced.pizza/user/lopter?tab=contributions)
<img width="323" alt="image" src="https://github.com/open-sauced/app/assets/34313413/4ee03d68-b675-4ecf-bfc6-4525666d80e1">

**Comment by jpmcb** [comment]
I _believe_ the discrepancy you're seeing here is because the algorithm on the front end only takes merged pull requests as the "last contribution date":

https://github.com/open-sauced/app/blob/d7c925b92795b18300032da3dad1a06ae6d57b00/components/molecules/ContributorListTableRow/contributor-list-table-row.tsx#L32-L34

If we wanted to include currently open PRs, we'd need to flip those to use `pr_created_at` vs. `pr_merged_at`

**Comment by BekahHW** [comment]
I think we should definitely be consistent here. 

I think there was another conversation about what was represented on a different issue. I think we need to make it clear and pick one. @jpmcb are you in favor of one over the other?
